### ISSUE TITLE
[Bug]: child bridge restart loop

### ISSUE THREAD

**Issue by takaiguchi** [homebridge-eufy-security/plugin#187]
### What happened?

log Shows it as restarting over and over again indefinitely. 

### Device Type

other (UnSupported)

### Plugin Version

beta (Best Effort)

### HomeBridge Version

v1.4.x (Supported)

### NodeJS Version

v16 (Supported)

### Operating System

Raspbian 10+ (Supported)

### Relevant log output

```shell
Can’t on my phone
```

### config

```shell
{
    "bridge": {
        "name": "Homebridge 877B",
        "username": "0E:11:DC:AC:87:7B",
        "port": 51416,
        "pin": "659-98-530",
        "advertiser": "avahi"
    },
    "accessories": [],
    "platforms": [
        {
            "name": "Config",
            "port": 8581,
            "auth": "form",
            "theme": "auto",
            "tempUnits": "f",
            "lang": "auto",
            "platform": "config"
        },
        {
            "platform": "EufySecurity",
            "username": "",
            "password": "",
            "country": "US",
            "ignoreStations": [
                "T8021P1020430AE9"
            ],
            "ignoreDevices": [],
            "_bridge": {
                "username": "0E:6E:16:75:73:AF",
                "port": 59271
            },
            "cleanCache": true,
            "enableDetailedLogging": false
        },
        {
            "name": "B-Hyve",
            "email": "",
            "password": "",
            "defaultRuntime": 1,
            "runtimeSource": 0,
            "useIrrigationDisplay": false,
            "displayValveType": 1,
            "showIrrigation": true,
            "showBridge": false,
            "showFloodSensor": false,
            "showTempSensor": false,
            "showLimitsSensor": false,
            "showStandby": false,
            "showRunall": false,
            "showSchedules": false,
            "_bridge": {
                "username": "0E:93:32:61:3E:AB",
                "port": 56339
            },
            "platform": "bhyve"
        },
        {
            "user": "",
            "password": "",
            "registeredApp": "MideaAir",
            "interval": 30,
            "devices": [
                {
                    "supportedSwingMode": "both",
                    "temperatureSteps": 1,
                    "minTemp": 17,
                    "maxTemp": 30,
                    "fanOnlyMode": false,
                    "OutdoorTemperature": true,
                    "useFahrenheit": false,
                    "audibleFeedback": false
                }
            ],
            "_bridge": {
                "username": "0E:1C:5E:BD:49:57",
                "port": 30502
            },
            "platform": "midea-air"
        },
        {
            "name": "Plugin Update",
            "sensorType": "contact",
            "platform": "PluginUpdate",
            "_bridge": {
                "username": "0E:58:8B:16:82:33",
                "port": 44410
            }
        }
    ],
    "disabledPlugins": []
}
```

**Comment by AndrewKite** [comment]
Firstly, I would go through your config posted and remove usernames and passwords. Maybe even update those passwords now for those services.

Secondly, I experienced a similar issue which was resolved by looking at the generated eufy log files which specified what country I was setup with. I am in Aus, and setup my Eufy account for Aus, but the logs displayed US. Once I setup the country to US in my config, my infinite reboot issue was resolved.

**Comment by AndrewKite** [comment]
@takaiguchi I ran into this issue again the other day. Different resolution this time.
I had an unsupported device (Bluetooth smart lock) which was causing the boot loop. As I have a separate account I share devices with from my main account, I was able to remove this unsupported device and get this plugin to launch successfully.
Double check the supported device list and compare to your own devices to make sure they are all accepted.
